src/progress_bar.rs

```diff
@@ -47,6 +47,18 @@ impl ProgressBar {
         Self::with_draw_target(Some(len), ProgressDrawTarget::stderr())
     }
 
+    /// Creates a new progress bar without a specified length
+    ///
+    /// This progress bar by default draws directly to stderr, and refreshes a maximum of 20 times
+    /// a second. To change the refresh rate, [set] the [draw target] to one with a different refresh
+    /// rate.
+    ///
+    /// [set]: ProgressBar::set_draw_target
+    /// [draw target]: ProgressDrawTarget
+    pub fn empty() -> Self {
```

Comment:
I think the `empty()` name isn't great for this. Maybe `no_length()`?

Reply:
Sure, no problem!